Update description.mdx formatting and content

ksen0 opened the pull request:
This is a very minor change to validate the translation tracker workflow. (Do not merge into main)

yinhwa commented:
mentioning @CodePsy-2001 here since he's expressed his interest in the previous /teach project and GSoC this year

montoyamoraga reviewed (approved):
hi @yinhwa thanks!
that was a lot of files because of the rebase :)
i went through your commits, and all your changes look great to me,
for next time could we split this into 2 PRs?
one for rebasing and one with your changes?
i guess that would be easier to review ~

lmccart commented:
hi @yinhwa could you close this PR and open a new one with just the files you edited?

yinhwa commented:
Thank you @montoyamoraga and @lmccart for checking in 😁😁 just one question to make sure before re-opening : should i just skip/eliminate the rebase part and only upload the relevant updated files? Much thanks 🙏🙏

lmccart commented:
Once you rebase, it should only show changes for the relevant files. If it shows more than that, yes please only add the relevant ones. Thanks!

yinhwa commented:
@lmccart got it thanks😁👍👍👍

ksen0 committed:
Update description.mdx formatting and content

This is a very minor change to validate the translation tracker workflow. (Do not merge.)
